metrics/cli/test/firebase/adapter/firebase_cli_service_adapter_test.dart

```diff
@@ -0,0 +1,444 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/firebase/adapter/firebase_cli_service_adapter.dart';
+import 'package:cli/firebase/cli/firebase_cli.dart';
+import 'package:cli/firebase/constants/firebase_constants.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../test_utils/matchers.dart';
+import '../../test_utils/prompter_mock.dart';
+
+void main() {
+  group('FirebaseCliServiceAdapter', () {
+    const projectId = 'projectId';
+    const workingDirectory = 'workingDirectory';
+    const target = FirebaseConstants.target;
+
+    final firebaseCli = _FirebaseCliMock();
+    final prompter = PrompterMock();
+    final firebaseService = FirebaseCliServiceAdapter(firebaseCli, prompter);
+    final stateError = StateError('test');
+
+    tearDown(() {
+      reset(firebaseCli);
+      reset(prompter);
+    });
+
+    test("throws an ArgumentError if the given Firebase CLI is null", () {
+      expect(
+        () => FirebaseCliServiceAdapter(null, prompter),
+        throwsArgumentError,
+      );
+    });
```

Comment:
```suggestion
    test(
      "throws an ArgumentError if the given Firebase CLI is null",
      () {
        expect(
          () => FirebaseCliServiceAdapter(null, prompter),
          throwsArgumentError,
        );
      },
    );
```
Please, apply such formatting to all other tests in this file to be consistent.